src/sage/rings/function_field/drinfeld_modules/drinfeld_module.py

```diff
@@ -1061,17 +1272,117 @@ def is_finite(self):
         from sage.rings.function_field.drinfeld_modules.finite_drinfeld_module import FiniteDrinfeldModule
         return isinstance(self, FiniteDrinfeldModule)
 
-    def j_invariant(self):
+    def is_isomorphic(self, psi):
         r"""
-        Return the j-invariant of the Drinfeld module if the rank is
-        two; raise a NotImplementedError otherwise.
+        Return ``True``  whether ``self`` is isomorphic to the Drinfeld module
+        `\psi`.
```

Comment:
As you probably understood by now, I think that explicitly naming parameters in explanations (`\psi`) is harder for the user, as it adds a layer to the explanation (imho).

        
```suggestion
        Return ``True``  whether ``self`` is isomorphic to the other
        Drinfeld module
```